src/js/languages/default.json

```diff
@@ -93,6 +93,22 @@
     "multiple": "multiple",
     "selected": "; Selected: {currentSelectedValue}"
   },
+  "selectMultiple": {
+    "clearAll": "Clear All",
+    "clearAllA11y": "{selected} options selected. Clear all?",
+    "open": "Open drop",
+    "optionNotSelected": "{optionLabel} not selected",
+    "optionSelected": "{optionLabel} selected",
+    "search": "Search to filter options",
+    "selectAll": "Select All",
+    "selectAllA11y": "Select all {total} options",
+    "selectDropDown": "Select dropdown",
+    "selected": "{selected} selected",
+    "selectedOfTotal": "{selected} selected of {total}",
+    "selectedOptions": "Selected options",
+    "showMore": "+ {remaining} more",
```

Comment:
Since `onMore` already has a special meaning in Grommet (InfiniteScroll `onMore`, Select `onMore` etc.) we thought it would be better to not use the `onMore` name here and instead change this to `showMore`

Reply:
thank you for clarifying